src/components/Events.js

```diff
@@ -1,4 +1,4 @@
-import style from "./Events.module.scss";
+import style from "../styles/Events.module.scss";
```

Comment:
Always keep the styles co-located with the component => that means in the same folder. It is much easier to find and one less thing to worry about when deleting the component

Reply:
done